src/main/java/world/bentobox/bentobox/listeners/PortalTeleportationListener.java

```diff
@@ -90,6 +90,14 @@ else if (plugin.getIslands().hasIsland(overWorld, e.getPlayer().getUniqueId())
             return true;
         }
         // TO END
+        
+        if (plugin.getIWM().isEndGenerate(overworld) && plugin.getIWM().isEndIslands(overworld) && plugin.getIWM().getEndWorld(overworld) != null) {
+            if (!plugin.getIslands().getIslandAt(e.getFrom()).hasEndIsland()) {
+                //To do, paste the schem/blueprint
+                
```

Comment:
Use something like this. You'll need to get the island object.
```
plugin.getIWM().getAddon(overworld).ifPresent(addon ->
new BlueprintPaster(plugin, getBlueprints(addon).get("end-island"), addon.getEndWorld(), island, null));
```

Reply:
I've already implemented it but I can't test it because BentoBox 1.5 doesn't load yet.

Reply:
What doesn’t load?

Reply:
`[13:58:34 ERROR]: Error occurred while enabling BentoBox v1.5.0-SNAPSHOT-b1112 (Is it up to date?)                      java.lang.ClassFormatError: Config classes must implement ConfigObject                                                          at world.bentobox.bentobox.database.yaml.ConfigHandler.<init>(ConfigHandler.java:23) ~[?:?]                             at world.bentobox.bentobox.database.yaml.YamlDatabase.getConfig(YamlDatabase.java:16) ~[?:?]                            at world.bentobox.bentobox.api.configuration.Config.<init>(Config.java:37) ~[?:?]                                       at world.bentobox.biomes.BiomesAddon.loadSettings(BiomesAddon.java:105) ~[?:?]                                          at world.bentobox.biomes.BiomesAddon.onLoad(BiomesAddon.java:24) ~[?:?]                                                 at world.bentobox.bentobox.managers.AddonsManager.loadAddon(AddonsManager.java:119) ~[?:?]                              at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source) ~[?:1.8.0_202]                                    at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:1.8.0_202]                                         at java.util.Spliterators$ArraySpliterator.forEachRemaining(Unknown Source) ~[?:1.8.0_202]                              at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:1.8.0_202]                                            at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:1.8.0_202]                                     at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source) ~[?:1.8.0_202]                              at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source) ~[?:1.8.0_202]                        at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:1.8.0_202]                                            at java.util.stream.ReferencePipeline.forEach(Unknown Source) ~[?:1.8.0_202]                                            at world.bentobox.bentobox.managers.AddonsManager.loadAddons(AddonsManager.java:72) ~[?:?]                              at world.bentobox.bentobox.BentoBox.onEnable(BentoBox.java:150) ~[?:?]                                                  at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:254) ~[patched_1.13.2.jar:git-Paper-535]                at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:326) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                          at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:406) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                         at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:443) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                           at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:357) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                          at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:236) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                             at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:786) ~[patched_1.13.2.jar:git-Paper-535]                                                                                                                              at java.lang.Thread.run(Unknown Source) [?:1.8.0_202]            `